go.mod

```diff
@@ -12,10 +14,10 @@ require (
 	github.com/hexops/autogold v1.3.0
 	github.com/iancoleman/strcase v0.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.143.0
-	github.com/pulumi/pulumi/sdk/v3 v3.143.0
+	github.com/pulumi/pulumi/pkg/v3 v3.150.0
```

Comment:
Would it be possible to update to [v3.151.0](https://github.com/pulumi/pulumi/releases/tag/v3.151.0) that includes https://github.com/pulumi/pulumi/pull/18618?